zoloto/coords.py

```diff
@@ -60,17 +60,17 @@ def rot_z(self) -> float:
     @property
     def yaw(self) -> float:
         """Get rotation angle around z axis in radians."""
-        return self.yaw_pitch_roll[0]
+        return self.yaw_pitch_roll[0]  # type: ignore
```

Comment:
wat

Reply:
Again, `cached_property`. I'm going to look into fixing this properly.

Reply:
e868eb5